scripts/validate.sh

```diff
@@ -97,33 +98,33 @@ function validate_shipyard_consumers() {
 }
 
 function validate_no_branch() {
-    if _git rev-parse "remotes/origin/${release['branch']}" >/dev/null 2>&1; then
+    if gh_commit_sha "${release['branch']}" >/dev/null 2>&1; then
         printerr "'${project}' already has stable branch '${release['branch']}'."
         return 1
     fi
 }
 
 function validate_project_commits() {
-    local project="${1:-${project}}"
-
-    clone_repo
+    local projects=("$@")
+    local latest_hash
 
-    if _git rev-parse "${release['version']}" >/dev/null 2>&1; then
-        printerr "'${project}' is already released with version '${release['version']}'."
-        return 1
-    fi
+    for project in "${projects[@]}"; do
```

Comment:
Given that the `projects` array isn’t used elsewhere, we could skip it and write

```suggestion
    for project; do
```

Reply:
Cool, didn't know bash had this feature